app/gui/src/dashboard/components/AriaComponents/Dialog/Dialog.tsx

```diff
@@ -13,6 +13,7 @@ import * as suspense from '#/components/Suspense'
 import * as mergeRefs from '#/utilities/mergeRefs'
 
 import { useDimensions } from '#/hooks/dimensionsHooks'
+import type { Spring } from '#/utilities/motion'
```

Comment:
@MrFlashAccount nit: i prefer
```ts
import { motion, type Spring } from '#/utilities/motion'
```

Reply:
I don't write it by hand, I rely on auto imports and eslint fixes the rest. I'm not opinionated on the imports, so I you prefer one over another, please adjust the eslint, if possible :)